### DISCUSSION TITLE
Soldering issues (THB1)

### DISCUSSION THREAD

**Discussion by AleXSR700** [pvvx/THB2#29] [Q&A]
Hi everyone,
I may not be the most experienced solderer, but I am able to solder.
But I find soldering the THB1 quite, how to put it, "fragile".

On one board the RX connection broke off and stuck to the cable rather than the board. On one the same happened with TX.

Does anybody have any idea if this can be prevented except with being ridiculously careful? Maybe using very flexible cables to prevent tension.
Or is it a heat problem? I found the solder didn't really take well until I heated the Weller to 380 °C.

And, most importantly, is there a way of re-soldering the two connections where the copper broke off? Scratch away at the board or something? 
Or would the only way be to connect to the chip directly?

Thanks for your feedback.
Alex

**Comment by pvvx** [comment]
380°C is too much for a PCB. 300°C is the maximum, otherwise the copper will peel off.

**Reply by AleXSR700** [reply]
Good to know :)

I had set it to 250 °C and it would not take. Tried 300 °C and it took but came lose more easily. So I tried 380 °C and got the connection.
But I should add that my tip was a bit too wide so I did not heat the copper on the PCB but rather the cable pin and added solder.
Nevertheless, probably too hot and too much stress.


To anybody interested:
I would not recommend using jumper cables (I think that is the translation) as seen in one of the issues.
Use one single copper wire of e.g. a piece of loudspeaker cable and then connect the jumper to those.

The thin copper wire is highly flexible and does not cause any strain.

Well, for the less experienced like me anyways.

Oh, and it one of the connectors breaks you can try to solder directly to the chip. I managed to do so on one out of two with broken connectors and it worked. On the second one the reset came lose and I must have shorted something trying to solder to that chip pin.
But RX and TX work.
GND and VCC don't matter because you can always solder to the battery connectors ;-)

**Reply by pvvx** [reply]
There are many options for 120 °C solder on the market.
I often flash the described devices without a soldering iron. You only need to hold the contacts with your hands for a few seconds...

**Reply by AleXSR700** [reply]
The solder itself melts way below 250 but it did not stick to the contacts.
Maybe too little flux in the solder. Will test with added flux if I ever have to solder these PCBs again.

**Reply by g6094199** [reply]
there is NOTHING like TO MUCH flux! "more is more" as we say in germany. then it will solder easily at 250°C. you will just have to remove the excess flux afterwards

**Comment by Bjk8kds** [comment]
This is what i do lately, because I don't have dupont cable/pin connector, I just using regular automotive cable (motorcycle wiring cable), but cut most of the strands of the cable (especially for the TX/RX pin since its too small), maybe only 1/4 or 1/5 left. Just hand soldering it to the board also to the USB TTL.
<details>
  <summary>picture</summary>

![Gambar WhatsApp 2024-09-19 pukul 11 35 20_9da8e4d4](https://github.com/user-attachments/assets/cb92a842-1a3a-402f-8c84-ef4b9905858d)
</details>
